contents/blog/gdpr-compliant-analytics.md

```diff
@@ -0,0 +1,83 @@
+* Data subjects can withdraw previously given consent whenever they want, and you have to honor their decision.
+* Children under 13 can only give consent with permission from their parent.
+* You need to keep documentary evidence of consent.
+
+So if you're tracking users in your product using PostHog to improve your product, you should explicitly ask for consent to use this data and explain exactly how you will use it when users sign up for your service.
+
+If you use PostHog with cookies on your website (for logged out users), you should also use a cookie banner to enable people to give and withdraw their consent for using cookies.
+
+**Security**
+
+You’re required to handle data securely by implementing “appropriate technical and organizational measures.”
+
+This means both technical measures (like encrypting data) and organizational measures (like staff training and limiting access to personal data).
+
+If you have a data breach, you have 72 hours to tell the data subjects or face penalties. (This notification requirement may be waived if you use technological safeguards, such as encryption, to render data useless to an attacker.)
+
+## How is GDPR compliance different with PostHog?
+
+PostHog is different from most other product analytics tools such as Mixpanel or Amplitude because it enables you to self-host on your own infrastructure and maintain full control of the data, this means you can decide on the technical and organizational security measures that are most appropriate for your organization. It also means that you can store your data within the EU, encouraging consent from EU citizens and making it easy to comply with any future data sovereignty requirements.
+
+## How to set PostHog up for GDPR compliant analytics
+
+### Step 1: Choose a hosting provider
+
+We recommend hosting PostHog on your own infrastructure, or a private cloud such as AWS, Google Cloud Platform or Microsoft Azure.
+
+### Step 2: Deploy PostHog
+
+Deploying PostHog onto your own infrastructure is very straightforward. You can follow our [standard deployment guides](https://posthog.com/docs/self-host) to get started, or [arrange a demo](https://posthog.com/book-a-demo) to see it in action first.
+
+### Step 3: Security configuration
+
+When setting up a PostHog instance **we strongly recommend that you use HTTPS** to secure data in transmission, whether or not your instance has access to the wider internet. We also have a [guide for securing PostHog](https://posthog.com/docs/self-host/configure/securing-posthog) which you should follow to further protect your instance.
+
+We also strongly recommend that you limit access to PostHog and the infrastructure it is deployed on only to people who are authorized and need to access the data, including shared dashboard links. Although aggregate data in dashboards should not contain Personal Data, it may be possible for malicious users to infer Personal Data unless it is evaluated thoroughly via expert determination.
```

Comment:
```suggestion
We also strongly recommend that you limit access to PostHog and the infrastructure it is deployed on only to people who are authorized and need to access the data, including shared dashboard links. Although aggregate data in dashboards should not contain personal data, it may be possible for malicious users to infer personal data unless it is evaluated thoroughly via expert determination.
```